src/resources/postcss/tickets/_modal.pcss

```diff
@@ -0,0 +1,314 @@
+		}
+
+		.tribe-modal__content {
+			.tribe-tickets__item {
+				-ms-grid-columns: 5fr 1fr 1fr 2fr;
+				grid-template-columns: 5fr 1fr 1fr 2fr;
+				padding-top: var(--spacer-3);
+
+				@media(--viewport-medium) {
+					-ms-grid-columns: 1fr 11fr 2fr 2fr 2fr;
+					grid-template-columns: 1fr 11fr 2fr 2fr 2fr;
+					padding-top: var(--spacer-2);
+				}
+
+				&.tribe-tickets__item--price-sufix {
+					@media(--viewport-medium) {
+						-ms-grid-columns: 1fr 8fr 5fr 2fr 2fr;
+						grid-template-columns: 1fr 8fr 5fr 2fr 2fr;
+					}
+				}
+			}
+
+			.tribe-tickets__item__total__wrap {
+				align-self: stretch;
+				display: flex;
+				justify-content: flex-end;
+				min-width: var(--spacer-8);
+				width: fit-content;
+				white-space: nowrap;
+			}
+
+			.tribe-tickets__item__total {
+				align-self: center;
+			}
+
+			.tribe-tickets__item__remove__wrap {
+				align-content: center;
+				align-self: stretch;
+				display: none;
+				width: var(--spacer-4);
+
+				@media(--viewport-medium) {
+					align-items: center;
+					display: flex;
+					-ms-grid-column: 1;
+					grid-column: 1;
+					-ms-grid-row: 1;
+					-ms-grid-row-span: 2;
+					grid-row: 1 / span 2;
+					justify-content: center;
+				}
+			}
+
+			.tribe-tickets__item__remove {
+				background-color: transparent;
+				background-image: svg-inline(remove-secondary);
+				background-position: center center;
+				background-repeat: no-repeat;
+				background-size: contain;
+				border-radius: 50%;
+				height: var(--spacer-4);
+				width: var(--spacer-4);
+
+				&:hover,
+				&:focus {
+					background-image: svg-inline(remove);
+					background-position: center center;
+					background-repeat: no-repeat;
+					background-size: contain;
+				}
+			}
+
+			.tribe-tickets__item__content__title {
+				@mixin heading-8;
+
+				-ms-grid-column: 1;
+				grid-column: 1;
+				-ms-grid-row: 1;
+				grid-row: 1;
+
+				@media(--viewport-medium) {
+					@mixin desktop-heading-6;
+
+					-ms-grid-column: 2;
+					grid-column: 2;
+				}
+
+				&.tribe-tickets--no-description {
+					align-self: center;
+					-ms-grid-row: 1;
+					-ms-grid-row-span: 2;
+					grid-row: 1 / span 2;
+					padding-top: 0;
+				}
+			}
+
+			.tribe-tickets__item__extra {
+				align-self: center;
+				display: flex;
+				flex-flow: column;
+				-ms-grid-column: 2;
+				grid-column: 2;
+				-ms-grid-row: 1;
+				-ms-grid-row-span: 2;
+				grid-row: 1 /span 2;
+				-ms-grid-columns:  auto;
+				grid-template-columns: auto;
+				-ms-grid-rows: auto;
+				grid-template-rows: auto;
+				text-align: center;
+
+				@media(--viewport-medium) {
+					-ms-grid-column: 3;
+					grid-column: 3;
+					margin-right: var(--spacer-1);
+				}
+
+				&.tribe-tickets__item__extra--price-suffix {
+					@media(--viewport-medium) {
+						display: inline-block;
+					}
+				}
+			}
+
+			.tribe-tickets__item__extra__price {
+				@mixin mobile-body-1;
+
+				@media (--viewport-medium) {
+					@mixin desktop-body-1-sup;
+				}
+			}
```

Comment:
if you add `tribe-common-b1` class to this element, you won't need these styles.

Reply:
I've been trying to trace where these things are taking place. 

Apparently these mods are like this because it's using the markup of the tickets block, from the modal and need adjustments in the font sizes. 

So, these elements already have `tribe-common-b2`, `tribe-common-b3` but the size should be different for **the modal**.

I'm afraid that it'll have to stay like this.

Reply:
👍